theme.config.jsx

```diff
@@ -1,5 +1,12 @@
 import { useRouter } from 'next/router'
-import { DocsThemeConfig } from 'nextra-theme-docs'
+import { DocsThemeConfig, LocaleSwitch } from 'nextra-theme-docs'
+
+const PLACEHOLDER_LOCALES = {
+  "en-US": "Search documentation",
+  fr: "Rechercher documents",
+  ru: "Поиск документации",
```

Comment:
Those two seem redundant, considering the commented out section below. Also, this is the only point there French comes up, that's probably an overlooked copy-paste.

Reply:
Ah, yeah, you're right, we don't need these language configs now, once we have the appropriate version we can add this back 👍